supplementary_style_guide/style_guidelines/mananged-services.adoc

```diff
@@ -0,0 +1,107 @@
+[[cloud-services]]
+= Cloud services
+
+Cloud services, also referred to as cloud services, are applications that run on a platform backed by data storage infrastructure and are hosted and managed by Red Hat on behalf of the customer. Managed cloud services provide rapid iteration and can more quickly deliver updated product functionality.
```

Comment:
+1, maybe "Cloud services, also referred to as managed cloud services," Since Marketing wants everyone to use 'cloud services' now instead of 'managed services', it would be good to use this term throughout and help socialize this more. But does 'cloud services' capture it all our purposes? 

Add'l info - from Nermina Miller, email Oct 19:
I want to alert you to a name change for what used to be known as managed services. To align with industry verbiage and help reduce confusion on the term "managed", Marketing has altered the names of our "Red Hat OpenShift managed cloud services" solution set and the "Red Hat managed cloud services" portfolio overall. 
Red Hat Cloud Services 
Includes all products that are managed by Red Hat including*: 
Red Hat OpenShift cloud services (i.e OSD, ARO, ROSA, RHOIC)
Red Hat OpenShift application services (i.e. RHOAM, RHOSAK, RHOSR)
Red Hat OpenShift data services (i.e. RHODS)
future cloud service solutions
Red Hat OpenShift cloud services
Previously "Red Hat OpenShift managed cloud services"  
Includes:
Red Hat OpenShift cloud services (i.e OSD, ARO, ROSA, RHOIC) 
Red Hat OpenShift application services
Includes:
Red Hat OpenShift application services (i.e. RHOAM, RHOSAK, RHOSR) 
Red Hat OpenShift data services
Includes:
Red Hat OpenShift data services (i.e. RHODS)
Roll-out: All new materials created will use the new naming and we will update existing materials in a rolling fashion. 
Our ask: Please start using this new terminology in your documents and presentations as we align as a company around the new naming structure. Also, please share this email with others on your team or other teams.

Reply:
@dayleparker @bergerhoffer I've removed "managed' entirely to align with the information Dayle shared.